examples/compare_connectivity_over_time_over_trial.py

```diff
@@ -0,0 +1,409 @@
+                       stim=False, eog=False, exclude='bads')
+
+# Create epochs for left visual field stimulus
+event_id, tmin, tmax = 3, -0.3, 1.6
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0))
+epochs.load_data()  # load the data
+
+###############################################################################
+# The sample data consist of repeated trials with a visual stimuli,
+# thus we use :func:`mne_connectivity.spectral_connectivity_epochs`
+# to compute connectivity over trials.
+
+sfreq = epochs.info['sfreq']  # the sampling frequency
+tmin = 0.0  # exclude the baseline period
+
+# To speed up the computation, we only use the first 5 epochs
+# and investigate the theta band
+epochs_sample = epochs[:5]
+
+Freq_Bands = {"theta": [4.0, 8.0]}
+n_freq_bands = len(Freq_Bands)
+min_freq = np.min(list(Freq_Bands.values()))
+max_freq = np.max(list(Freq_Bands.values()))
+
+# Prepare the freq points
+freqs = np.linspace(min_freq, max_freq, int((max_freq - min_freq) * 4 + 1))
+
+fmin = tuple([list(Freq_Bands.values())[f][0] for f in range(len(Freq_Bands))])
+fmax = tuple([list(Freq_Bands.values())[f][1] for f in range(len(Freq_Bands))])
+
+# We specify the connectivity measurements
+connectivity_methods = ["wpli"]
+n_con_methods = len(connectivity_methods)
+
+# Compute connectivity over trials
+con_epochs = spectral_connectivity_epochs(epochs_sample,
+                                          method=connectivity_methods,
+                                          sfreq=sfreq, mode="cwt_morlet",
+                                          cwt_freqs=freqs, fmin=fmin,
+                                          fmax=fmax, faverage=True,
+                                          tmin=tmin, cwt_n_cycles=4)
+
+###############################################################################
+# Notice we have shortened the wavelets to 4 cycles since we only have 1.6s
+# epochs and are looking at theta activity. This might make the connectivity
+# measurements more sensitive to noise.
+
+# Plot the global connectivity over time
+n_channels = epochs.info["nchan"]  # get number of channels
+times = epochs.times[epochs.times >= tmin]  # get the timepoints
+n_connections = (n_channels * n_channels - n_channels) / 2
+
+# Get global avg connectivity over all connections
+con_epochs_raveled_array = con_epochs.get_data(output="raveled")
+global_con_epochs = np.sum(con_epochs_raveled_array, axis=0) / n_connections
+
+# Since there is only one freq band, we choose the first dimension
+global_con_epochs = global_con_epochs[0]
+
+fig = plt.figure()
+plt.plot(times, global_con_epochs)
+plt.xlabel("Time (s)")
+plt.ylabel("Global theta wPLI over trials")
+
+# Get the timepoint with highest global connectivity
+t_con_max = np.argmax(global_con_epochs)
+
+# Plot the connectivity matrix at the timepoint with highest global wPLI
+con_epochs_array = con_epochs.get_data(output="dense")
+
```

Comment:
Perhaps in line w/ https://github.com/mne-tools/mne-connectivity/pull/129/files#r1128169084, we can then add some comments about how the highest time point connectivity means something perhaps?

Is it correct to say that we are "extracting the timepoint at which there is the most correlation in the theta frequency band as measured by wPLI (or some other connectivity metric) and visualizing the connectivity"?

Maybe we can also mention something about the output of the final connectivity matrix. I.e. are the channels in occipital cortex the ones with generally higher correlation?

Reply:
I added a bit in the conclusion and also the plotted the sensor connectivity, to visualize the 20 highest connections which was between the occipital and frontal areas.